kanidmd/src/lib/server.rs

```diff
@@ -2340,6 +2289,7 @@ impl<'a> QueryServerWriteTransaction<'a> {
         } else {
             ladmin_error!(audit, "initialise_schema_core -> Error {:?}", r);
         }
+        // why do we have error handling if it's always supposed to be `Ok`?
```

Comment:
Because I suspect there was a time when this WASNT returinng ok and so this allowed me to find that error faster during test/dev rather than the error bubbling out?